salesforce_api/mysql-record-to-sfdc-new-product/readme.md

```diff
@@ -0,0 +1,49 @@
+Example to create a new product in Salesforce using MySQL and Salesforce integration.
```

Comment:
Be consistent on file name. Some have `Readme.md` while this is`readme.md`

Reply:
Updated in [c20653b](https://github.com/ballerina-guides/integration-samples/pull/31/commits/c20653bca0edc3a9b9e7004e44d9128684c26348)